app/controllers/api/v1/tickets_controller.rb

```
@@ -1,311 +1,424 @@
+        scope = scope.where(status: params[:status]) if params[:status].present?
+        scope = scope.where(ticket_type: params[:ticket_type]) if params[:ticket_type].present?
+        scope
+      end
+
+      def set_ticket
+        @ticket = @organization.tickets.find_by!(ticket_number: params[:id])
+      rescue ActiveRecord::RecordNotFound
+        render json: { error: 'Ticket not found' }, status: :not_found
+      end
+
+      def ticket_params
```

RuboCop (1.73)

[warning] Method `Api::V1::TicketsController#ticket_params` is defined at both app/controllers/api/v1/tickets_controller.rb:230 and app/controllers/api/v1/tickets_controller.rb:284.

(Lint/DuplicateMethods)